client/plrdlg_common.cpp

```diff
@@ -436,8 +436,9 @@ QString col_research(const struct player *them)
     return _("-");
   } else if (BV_ISSET(them->client.visible, NI_TECHS)) {
     struct research *research = research_get(them);
-    return research_advance_name_translation(research,
-                                             research->researching);
+    return research_advance_name_translation(research, research->researching)
+           + " " + "(" + QString::number(research->bulbs_researched) + "/"
+           + QString::number(research->client.researching_cost) + ")";
```

Comment:
Avoid building strings this way, it makes it harder to translate if needed later. Prefer the string-building function `arg`:
```suggestion
    return QStringLiteral("%1 (%2/%3)").arg(research_advance_name_translation(research, research->researching))
           .arg(research->bulbs_researched).arg(research->client.researching_cost);
```
(with proper `clang-format`ting)